webgl.go

```diff
@@ -1,6 +1,7 @@
 // +build wasm
 
-// Copyright 2014 Joseph Hager and the TinyGo Authors. All rights reserved.
+// Copyright 2020 Erin Pentecost, Joseph Hager and the TinyGo Authors.
```

Comment:
Might as well add my name in there too.  It's not something I generally care about, but since you're already at it, it feels kind of wrong to exclude myself. :wink: